crates/parser/src/grammar/patterns.rs

```diff
@@ -214,32 +245,26 @@ fn record_pat_field_list(p: &mut Parser) {
     while !p.at(EOF) && !p.at(T!['}']) {
         match p.current() {
             // A trailing `..` is *not* treated as a REST_PAT.
-            T![.] if p.at(T![..]) => p.bump(T![..]),
+            T![.] if p.at(T![..]) => {
+                rest_pat(p);
+            }
             T!['{'] => error_block(p, "expected ident"),
+            T![#] => {
+                let m = p.start();
+                attributes::outer_attrs(p);
```

Comment:
Hm, seems like we can move this `outer_attrs` call to before the `match`? like we do in `record_expr_field_list`

Reply:
Yes, the code will be cleaner.